### PR TITLE
Make comment section present on all pages, add tooltip text.

### PR DESCRIPTION
**Pull request by ganewman**
Add HTML to all web pages in order to make comment page present (and the same) on all pages.
Also, add tooltip/hover text which shows the time comments were made.
Add some additional TODOs for if time allows later.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -79,13 +79,18 @@ function createCommentDiv(commentObj) {
...
+  tooltipElement.innerText = commentObj.time;
+  nameElement.appendChild(tooltipElement);
   commentDiv.appendChild(nameElement);
 
   const contentElement = document.createElement('p');
   contentElement.innerText = commentObj.content;
   commentDiv.appendChild(contentElement);
 
-  // TODO: do something useful with the time information
+  // TODO: Convert from fairly useless MS time to hour/minute.
```
Comment (kellyktan) [inline portfolio/src/main/webapp/script.js:93]: Haha was just about to put a comment above with this suggestion! Could be good to include the date as well as the hour/minute (plus timezone (feel free to just use eastern time to make it easier))

Comment (ganewman) [reply portfolio/src/main/webapp/script.js:93]: I definitely would like to. I looked into it a little bit and it seems like there's no good library function to convert, so I would have to write a conversion function myself. I might add it after I've made some progress on the week 4 walkthrough.

Comment (kellyktan) [reply portfolio/src/main/webapp/script.js:93]: Sounds good!

File: portfolio/src/main/webapp/about.html [hunk 2]
```diff
@@ -70,7 +70,24 @@ <h3 id="pets">Pets</h3>
...
       </p>
-      <h3>
+      <!-- TODO: Find a way to make comments and navbar common to all pages
+          without actually repeating HTML code, as it is now. -->
+      <div id="comments">
+        <h3>Comments</h3>
+        <form action="/data" method="POST">
+        Name:
+        <br/>
+        <input type="text" name="name-input" />
```
Comment (suparnabarua) [inline portfolio/src/main/webapp/about.html:80]: Can you indent the tags that are inside of the form action tag

File: portfolio/src/main/webapp/gallery.html [hunk 3]
```diff
@@ -27,6 +27,24 @@ <h1>Image Gallery</h1>
...
             dipping me at a competition." id="tufts-standard" 
             onclick="displayCaption(this)"/>
+      <!-- TODO: Find a way to make comments and navbar common to all pages
+          without actually repeating HTML code, as it is now. -->
+      <div id="comments">
+        <h3>Comments</h3>
+        <form action="/data" method="POST">
+        Name:
+        <br/>
+        <input type="text" name="name-input" />
```
Comment (suparnabarua) [inline portfolio/src/main/webapp/gallery.html:37]: Fix the indentation here as well

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -79,13 +79,18 @@ function createCommentDiv(commentObj) {
 
   const nameElement = document.createElement('h4');
   nameElement.innerText = commentObj.name;
+  nameElement.className = "tooltip";
+  const tooltipElement = document.createElement('span');
```
Comment (suparnabarua) [inline portfolio/src/main/webapp/script.js:83]: rename this to toolTip

File: portfolio/src/main/webapp/about.html [hunk 5]
```diff
@@ -70,7 +70,24 @@ <h3 id="pets">Pets</h3>
...
           happy kitty! There are some pictures of him on the gallery page.
       </p>
-      <h3>
+      <!-- TODO: Find a way to make comments and navbar common to all pages
+          without actually repeating HTML code, as it is now. -->
+      <div id="comments">
+        <h3>Comments</h3>
+        <form action="/data" method="POST">
+          Name:
+          <br/>
```
Comment (kellyktan) [inline portfolio/src/main/webapp/about.html:79]: nit: should be `<br>`  not `<br />`
https://google.github.io/styleguide/htmlcssguide.html#Document_Type

**Pull request merged**